airbyte-cdk/python/airbyte_cdk/base_python/cdk/streams/auth/oauth.py

```diff
@@ -34,7 +34,7 @@ class Oauth2Authenticator(HttpAuthenticator):
     The generated access token is attached to each request via the Authorization header.
     """
 
-    def __init__(self, token_refresh_endpoint: str, client_id: str, client_secret: str, refresh_token: str, scopes: [str] = None):
+    def __init__(self, token_refresh_endpoint: str, client_id: str, client_secret: str, refresh_token: str, scopes: List[str] = None):
```

Comment:
Had to guess this from looking at code, does this look right?

Reply:
yes, it's probably why it's imported from a package named `typing` to be more explicit (both are valid `List`)